nipype/interfaces/io.py

```diff
@@ -2721,18 +2721,25 @@ def _list_outputs(self):
 
 
 class BIDSDataGrabberInputSpec(DynamicTraitedSpec):
-    base_dir = Directory(exists=True,
-                         desc='Path to BIDS Directory.',
-                         mandatory=True)
-    output_query = traits.Dict(key_trait=Str,
-                               value_trait=traits.Dict,
-                               desc='Queries for outfield outputs')
-    raise_on_empty = traits.Bool(True, usedefault=True,
-                                 desc='Generate exception if list is empty '
-                                 'for a given field')
-    return_type = traits.Enum('file', 'namedtuple', usedefault=True)
-    strict = traits.Bool(desc='Return only BIDS "proper" files (e.g., '
-                              'ignore derivatives/, sourcedata/, etc.)')
+    base_dir = Directory(
+        exists=True,
+        desc='Path to BIDS Directory.',
+        mandatory=True)
+    output_query = traits.Dict(
+        key_trait=Str,
+        value_trait=traits.Dict,
+        desc='Queries for outfield outputs')
+    raise_on_empty = traits.Bool(
+        True, usedefault=True,
+        desc='Generate exception if list is empty for a given field')
+    return_type = traits.Enum(
+        'file', 'namedtuple', usedefault=True)
```

Comment:
Have we tested `return_type == 'namedtuple'` anywhere? I think this went away in 0.7.0. If we're going to drop backwards compatibility, it might be worth removing this one.

Reply:
I'll remove that